### ISSUE TITLE
espeakup cannot be used in termux on Android platform

### ISSUE THREAD

**Issue by w-sx** [linux-speakup/espeakup#53]
termux" is a Linux environment emulator on the Android platform that allows you to use tools such as ssh and Python in it. However, termux does not support Android's screen reader very well.
Therefore, I thought of whether I could solve the accessibility problem by installing "espeakup" in termux. However, "espeakup" cannot be installed via termux's package manager "pkg".
Fortunately, I was able to install and run the Debian environment through termux's "proot-distro" command, and I was able to install "espeakup" smoothly in the Debian environment.
However, "espeakup" cannot run normally. Using 'espeakup -d' reports 'Unable to open the softsynth device: No such file or directory' error.
Can this problem be solved? Or can you give me some guidance? Or are there any other command-line screen readers available?

**Comment by w-sx** [comment]
I have run the 'espeak' command in both termux's own command line and termux's Debian environment, and I have heard the sound of speech synthesis gas, but 'espeakup' still cannot be started.

**Comment by alex19EP** [comment]
espeakup is just a connector to speakup witch is a inkernel screen reader.

kernels used on android phones does not compile modules for speakup. even if the module were built, then loading it requires root, and is generally useless.

**Comment by fcarmickle** [comment]
Since you have python, try Fenrir.
